src/utils.js

```diff
@@ -119,55 +119,76 @@ module.exports = class Utils {
   static selectFiles(config = {}) {
     return new Promise((resolve, reject) => {
       /**
-       * Create a new INPUT element
-       * @type {HTMLElement}
+       * Check
        */
-      let inputElement = document.createElement('INPUT');
-
-      /**
-       * Set a 'FILE' type for this input element
-       * @type {string}
-       */
-      inputElement.type = 'file';
+      if (input === null) {
```

Comment:
![image](https://user-images.githubusercontent.com/15259299/102115581-cea09880-3e4c-11eb-87a4-5b771ff05a30.png)

Reply:
Oops, sorry.
I commited aboute this.